rules.md

```diff
@@ -511,140 +422,177 @@ Ultimately, decisions on whether some output meets the requirements specified in
- - Mediate updates to these Rules
- - Any other actions to enforce adherence to these rules
+Each Strategist submits quarterly proposals to the Lead Strategist to increase their budgets.  The content and purpose of this proposal is similar to the Lead Strategist's Network Proposal.  It should report project updates, value propositions, budget spending, quarterly priorities, and anything else the strategist wants to commmunicate in order to help justify the value delivered to the network for the funding received and requested.
 
- If the Proposal Owner needs to exercise any of these rights, they must leave a comment (e.g. on Trello or Github depending on the action) detailing the action and explaining the reasons as to why an action was taken.
+## 5.2 Budgets
 
-The Proposal Owner can be replaced through a vote for a new Admin to be the PO as per the voting criteria defined in Section 4.5 - Governance.  If that occurs, the newly elected PO will assume all PO responsibilities and the existing PO will transfer all funds from the Proposal Wallet to a wallet controlled by the new PO.
+There are two types of budgets in the Incuabtor:
 
-> The Proposal Owner role is incentivized by a [Service Bounty](https://trello.com/c/pjOIKxyc/100-dash-incubator-proposal-owner) defined within the Incubator app itself.
+- The Incubator general budget
+- Individual stategist budgets
 
-## 5.4 Auditing
+### 5.2.1 General Budget
 
-As an [Open-source DFO](#14-full-transparency), anyone can perform a full public audit of the Incubator at any time, to check that all work is being conducted as per these Rules, and verify the rewards issued to individual Incubator users for that work via the Dash blockchain.
+Network proposal funds are received to the published [Incubator Wallet](#63-accounting) address(es) and pooled across proposals as a rolling budget available to Strategist budgets.
 
-Below we provide guidelines on how to audit the Incubator from a financial and work perspective, but people can devise their own audit strategies as they deem fit.
+The Incubator is a pure Dash fund meaning that rewards are denominated only in Dash.  Therefore any fluctuations in the fiat price of Dash will only reflect the pricing of new [Tasks](#25-tasks) by Admins and prior agreements made in Dash are not necessarily renegotiated or adjusted.
 
-### 5.4.1 Auditing Accounts
+### 5.2.2 Strategist Budgets
 
-Every Dash of funding spent by the Incubator must be accounted for via a [Task Claim](#26-claims) approved by an [Admin](#4-admins) with an associated transaction on the Dash blockchain.
+Approved Strategist proposal funds are sent by the Lead Strategist to a distinct address for each Strategist.  These addresses are currently controlled by the Lead Strategist, but when the system matures and Strategists agree to it, the Lead Strategist will instead send funds to addresses each Strategist controls and these rules and processes will be updated accordingly.
```

Comment:
technicality - this will be a rule change and subject to Governance vote or consensus of Strategists? May want to describe that explicitly.